changelog/index.php

```diff
@@ -75,15 +76,26 @@ protected function execute(InputInterface $input, OutputInterface $output)
 				"The provided format is invalid (should be one of markdown, forum, html but was '$format')"
 			);
 		}
+
+		$repoName = $input->getArgument('repo');
 		$base = $input->getArgument('base');
 		$head = $input->getArgument('head');
 
+		$output->writeln("repo: $repoName");
 		$output->writeln("base: $base");
 		$output->writeln("head: $head");
 
 		$milestoneToCheck = null;
-		if (substr($base, 0, 1) === 'v') {
-			$version = explode('.', substr($base, 1));
+		$substring = 'v';
+		$subStringNum = 1;
+		if(strcasecmp($repoName, $server) != 0 ){
```

Comment:
```suggestion
		if($repoName !== $server){
```

This should be enough. Even if this is not case sensitive.

Reply:
Done.